vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
Okay, the impression I got from that reply was that it wasn't important that the enum values be unique as a differentiator.  The issue @jktomer raised is one of the reasons why I was [suggesting](https://github.com/votinginfoproject/vip-specification/issues/30#issuecomment-93985805) the idea to have a district type object (as opposed to enum).  It would be guaranteed to be unique, in addition to capturing more of the relationships.